packages/form-builder/src/common/getWindowData.ts

```diff
@@ -6,10 +6,21 @@ declare global {
             formDesigns: FormDesign[];
             formPage: FormPageSettings;
             currency: string;
-        };
+        },
+        formBuilderData?: {
+            gateways: Array<any>;
+        },
     }
 }
 
 export default function getWindowData() {
     return window.storageData;
```

Comment:
The concept of storage data needs to be refactored as the scope has changed significantly over time.